packages/whale-api-client/src/api/RawTx.ts

```diff
@@ -0,0 +1,40 @@
+import { RawTx } from '@defichain/ocean-api-client'
```

Comment:
a note for my own
https://github.com/DeFiCh/jellyfish/pull/949
https://github.com/DeFiCh/jellyfish/pull/948